app/views/content/landing/home.md

```diff
@@ -0,0 +1,18 @@
+---
+  layout: "layouts/home"
+  title: "Get Into Teaching"
+  description: |-
+    Read official Department for Education information to help you become a teacher. Learn about funding, teacher training, and steps to become a teacher. 
+  fullwidth: true
+  title_paragraph: "Whether you're just thinking about it or you're ready to apply, nobody knows teaching like we do. We offer free advice and support to decide if teaching in a primary or secondary school in England is right for you."
```

Comment:
```suggestion
  title_paragraph: "Whether you're just thinking about teaching or you're ready to apply, we offer free advice and support to decide if teaching in a primary or secondary school in England is right for you."
```
I'm not sure if I think this first sentence flows super well? Whilst I know 'Nobody knows teaching like we do' is one of the things in the service proposition we wanna try, I'm not sure it feels right in this paragraph? Doesn't seem to fit super well with the gist of what we're saying, if that makes sense?

@philspray

Reply:
Could we not just switch it around to say 'Nobody knows teaching like we do. Whether you're just thinking......, we offer free advice......'?

@philspray @EllieNodder